test/integration/webhook/v1alpha2/resourceflavor_test.go

```diff
@@ -76,6 +83,43 @@ var _ = ginkgo.Describe("ResourceFlavor Webhook", func() {
 		})
 	})
 
+	ginkgo.DescribeTable("invalid number of properties", func(taintsCount int, nodeSelectorCount int, isInvalid bool) {
+		resourceFlavor := func() *kueue.ResourceFlavor {
```

Comment:
this anonymous function is unnecessary.

Anonymous functions might be necessary when the table accepts a spec, instead of parameters.
https://github.com/kubernetes/kubernetes/blob/c1c0e4fe0bb4e7c0145d45a010577ed64619903a/pkg/apis/batch/validation/validation_test.go#L285-L305